src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,37 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecateDefaultImage.
+     * @throws IOException In case of error.
+     * @todo @todo #754:30min Implement a deprecation message at the start
+     *  of the process if the project is using the default image
+     *  'yegor256/rultor'.
```

Comment:
@sebing can we join these puzzles? They are pretty much related and shouldn't be done in parallel